docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[release-notes-header-8.4.0]]
+== 8.4
+
+[discrete]
+[[release-notes-8.4.0]]
+=== 8.4.0
+
+[discrete]
+[[known-issue-8.4.0]]
```

Comment:
We have a known issue related to the prebuilt rule upgrade process: https://github.com/elastic/kibana/issues/139095. Should we also mention it here?

Reply:
@xcrzx thanks for bringing this to our attention. If the bug affects `8.4.`, yes we can add that here. :)

Reply:
I'd summarize it into something like this: A banner saying that an update is available for prebuilt detection rules is being erroneously displayed even when all rules are up to date. And unfortunately, there is no workaround to hide the banner. The issue will be fixed in `8.4.1`.

Reply:
Thanks, @xcrzx ! We don't normally make forward-looking comments like when a bug will be fixed, but we can definitely use the first part and, if needed, mention that there's no current no workaround for removing the banner.